hacking/shippable/rebalance.py

```diff
@@ -86,15 +91,18 @@ def get_raw_test_targets(args, test_path):
                 continue
 
             info = json.loads(test['contents'])
+            targets = info.get('targets')
 
-            for target_name, target_info in info.get('targets', {}).items():
-                target_runtime = int(target_info.get('run_time_seconds', 0))
+            targets = info.get('targets')
+            if isinstance(targets, dict):
```

Comment:
I could also catch and handle the exception if that's better than checking the type.

Reply:
What is the type in the scenario you've seen? It sounds like `targets` is something and we shouldn't be ignoring it?

Reply:
> What is the type in the scenario you've seen?

It's a list. I put an example of the data it is stack tracing on in the issue description.

>It sounds like targets is something and we shouldn't be ignoring it?

I was thinking that too. But when `targets` is a list, we don't have any of the other data we need, namely `run_time_seconds`. I didn't know what else to do besides ignore it. I suppose we could at least list the target name and just `0` or the run time, but that's an inaccurate representation.

The bigger question I have is why are we sometimes getting only a list of targets with no job data?